src/extension/extensionServer.ts

```diff
@@ -235,6 +237,16 @@ export class ExtensionServer implements vscode.Disposable {
                 TargetPlatformHelper.checkTargetPlatformSupport(mobilePlatformOptions.platform);
                 return mobilePlatform.beforeStartPackager()
                     .then(() => {
+                        generator.step("getReactNativeVersion");
+                        return ReactNativeProjectHelper.getReactNativePackageVersionFromNodeModules(
+                            path.resolve(request.arguments.cwd, "node_modules", "react-native")
```

Comment:
I think we can avoid path resolving if we start to use `package.dependencyPackage()` function from the project root.
[Example](https://github.com/microsoft/vscode-react-native/pull/1093/files#diff-164a18e7473c9450518245cd953bc97fR41)

Reply:
Thanks for your advice, this is a good idea. I'll try to implement such approach.